Makefile

```diff
@@ -58,32 +56,30 @@ copycomp:
 
 # Build api docs
 updateapispec:
-	cp $(K8SROOT)/api/openapi-spec/swagger.json gen-apidocs/generators/openapi-spec/swagger.json
+	cp $(K8SROOT)/api/openapi-spec/swagger.json gen-apidocs/config/swagger.json
 
 api: cleanapi
-	go run gen-apidocs/main.go --config-dir=gen-apidocs/generators --munge-groups=false
+	go run gen-apidocs/main.go --config-dir=gen-apidocs --munge-groups=false
 
-# NOTE: The following "sudo" may go away when we remove docker based api doc generator
 cleanapi:
-	sudo rm -rf $(shell pwd)/gen-apidocs/generators/build
-	sudo rm -rf $(shell pwd)/gen-apidocs/generators/includes
+	rm -rf $(shell pwd)/gen-apidocs/build
 
 copyapi:
 	cp $(APISRC)/build/index.html $(APIDST)/index.html
 	cp $(APISRC)/build/navData.js $(APIDST)/navData.js
-	cp $(APISRC)/static/scroll.js $(APIDST)/scroll.js
+	cp $(APISRC)/config/static/scroll.js $(APIDST)/scroll.js
```

Comment:
nit: why is `scroll.js` considered configuration?

Reply:
em, it is a terminology confusion I think. I don't see a clear answer for the best location of the static files. We can move it out of the `config` subdirectory, but that is still confusing. If the `config` directory name is not good, do you have a better option?

Reply:
Hi @tengqm. Since the files are copied to k8s/website, the directory location is not as important.
Why is this change needed? Just to clean up the directory hierarchy?
The directory names help `document` the contents.
When a new theme is in place, these static files may not be needed, or just some may be needed.
You could leave the files in the static directory or the static files could be placed in a directory structure such as:
```
gen-apidocs
    |-- static/              Directory for static files.
    |    -- css/
    |    -- fonts/
    |    -- js/
```

Reply:
Okay, let's try your way and see if it looks better.
Yes, the intent behind this PR is to clean up the directory hierarchy. As a developer, I myself sometimes get confused by the files/directories when trying to fix things. :)